zhcsshim.go

```diff
@@ -48,6 +45,28 @@ var (
 	procTerminateComputeSystem                     = modvmcompute.NewProc("TerminateComputeSystem")
```

Comment:
Maybe you should just remove the old APIs in one swoop so that it's clean when it's vendored into docker and libcontainerd uses the new APIs.